kafe/fit/tools/ensemble.py

```diff
@@ -436,9 +436,10 @@ class EnsembleVariablePlotter(object):
     """
 
     _DEFAULT_PLOT_PDF_KWARGS = dict(marker='')
-    _DEFAULT_PLOT_EXPECTED_MEAN_KWARGS = dict(linewidth=1, marker='', linestyle='--',
```

Comment:
As mentioned in the OP, I changed these lines because they cause my laptop setup to crash (`matplotlib` 1.5.1).
The intent is to revert this change before merging.
What do you have in mind for a compatibility check?
A simple try/except?
What would be a good fallback value?

Reply:
OK, sounds good. I think a simple try/except should be sufficient.
With the old matplotlib API, `mpl.rcParams['axes.prop_cycle']` should give a `KeyError`,
so you could then get the color from `mpl.rcParams['axes.color_cycle']` instead.